src/test/java/net/dries007/tfc/test/drawing/BiomeNoiseTest.java

```diff
@@ -42,30 +41,48 @@ public class BiomeNoiseTest extends TestHelper
     @Test
     public void testRollingHills()
     {
-        terrain.draw("noise_rolling_hills", BiomeNoise.hills(seed(), -5, 28));
+        terrain.draw("noise_rolling_hills", hills(TestSetup.seed(), -5, 28));
```

Comment:
By modifying the implementation to be `default` instead of `static`, means we don't have to quality all these calls in tests.

```suggestion
        terrain.draw("noise_rolling_hills", hills(seed(), -5, 28));
```